### PR TITLE
Add rustls security review

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
@scovetta or @Amir-Montazery - Okay to merge?

**Comment by david-a-wheeler** [comment]
Hmm, getting these errors, I don't understand why:

* Error: reviews/rustls/review-1.md: Missing Package URL.
* Error: reviews/rustls/review-1.md: Invalid scope.

**Comment by scovetta** [comment]
Sorry, took a while to get to this, but some of the fields were mis-spelled, fixed.

**Review by DIEGOHJJACOBO — COMMENTED** [review]
[f330b89336d3654daed15e03164c054a9e95eef5]

File: reviews/rustls/review-1.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
```
Comment (DIEGOHJJACOBO) [inline reviews/rustls/review-1.md:1]:
```suggestion
---[f330b89336d3654daed15e03164c054a9e95eef5]
```

**Review by DIEGOHJJACOBO — COMMENTED** [review]
[f330b89336d3654daed15e03164c054a9e95eef5]

**Pull request merged**